fennel/lib/to_proto/source_code.py

```diff
@@ -40,8 +50,36 @@ def get_featureset_core_code(featureset: Featureset) -> str:
     # If python version 3.8 or below add @feature decorator
     if sys.version_info < (3, 9):
         source_code = f"@featureset\n{dedent(source_code)}"
-    z = _remove_empty_lines(source_code)
-    return z
+    return _remove_empty_lines(source_code)
+
+
+def get_featureset_gen_code(
```

Comment:
Is this logic tested via some unit tests? I'm not seeing any tests where we don't sync a few indirect dependencies and get an error back?

Reply:
yeah tests were failing and i fixed those :p let me create invalid test cases